<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="home">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>33</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¡ Listo para empezar !</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mensage_home">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscador">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingresa el nombre o el ID de un producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_buscar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_id">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_nombre">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar producto por:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_producto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="cantidad_producto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
/* Estilos base para el botón */
QPushButton{
  display: inline-block;
  padding: 10px 20px;
  background-color: none;
  color: blue;
  border: none;
  border-radius: 20px; /* Bordes redondeados */
  cursor: pointer;
  transition:color 0.3s; /* Animación de cambio de color */
}

QPushButton:hover {
  color: rgb(5, 125, 245); /* Color más oscuro al pasar el cursor */
}</string>
    </property>
    <property name="text">
     <string>Cerrar Sesión</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
